.github/workflows/pr-to-main.yml

```diff
@@ -0,0 +1,42 @@
+name: Merge staging Into Main
+
+on:
+  push:
+    branches:
+      - 'staging' #to update to staging
+
+permissions:
+  pull-requests: write
+
+jobs:
+  main:
+    name: Create PR Release to Main
+    runs-on: ubuntu-latest
+    steps:
+      - name: git checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+      
+      - name: get pr name
+        run: |
+          echo "$GITHUB_CONTEXT"| jq '.event.commits[].id' | tail -2 | head -1 |sed 's/\"//g'
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+    
+      # https://github.com/marketplace/actions/github-pull-request-action
+      - name: create pull request with reposync action2
```

Comment:
is it possible to add the original PR authors to the merge commit in main?

Reply:
thats a great point, we need to consider this for release notes

Reply:
i m getting some testing in my fork to see how the commits looks after they merge into main. I m hoping to preserve the original commit by the author.

Reply:
HI @akashsinghal , here are the sample PR with sequence of action.
1. Dev opens a PR from "myBranch" to staging 
[feat: title Apr 3 PR1 by susanshi · Pull Request #283 · susanshi/ratify (github.com)](https://github.com/susanshi/ratify/pull/283)

2. Test passes and Maintainer approves the PR
3. Dev/Maintainer merges the PR with commit title
4. An automated PR created form push to staging
https://github.com/susanshi/ratify/pull/284 , the title is the commit title for the previous PR.

5. Full test validation is required, and maintainer manually merge the PR.  From the PR, I could see the author of the original commit, there should be an option to add the original author on manual merge.

![image](https://github.com/deislabs/ratify/assets/8160956/43f57335-bf93-40d4-ab56-1f1e0d640414)